graph_test_set/sequence_file_linkto_cell_suspesion.adoc

```diff
@@ -13,6 +13,7 @@ This next query provides the filenames, biomaterial IDs and nucleic acid sources
 ----
 MATCH path = (b:biomaterial)-[:INPUT_TO_PROCESSES]->(p:process)<-[:DERIVED_BY_PROCESSES]-(f:sequence_file), seq_path = (p)-[:PROTOCOLS]-(l:library_preparation_protocol)
 WHERE NOT b:cell_suspension
+AND NOT b:imaged_specimen
 AND l.nucleic_acid_source STARTS WITH "single"
```

Comment:
```suggestion
```

This was added for bulk datasets, but we have now decided that bulk datasets will have cell suspensions so we can delete this!